examples/combined_ODNP.py

```diff
@@ -0,0 +1,343 @@
+from numpy import *
+from pyspecdata import *
+import os
+import sys
+import SpinCore_pp
+import time
+from Instruments import power_control
+from datetime import datetime
+from SpinCore_pp.verifyParams import verifyParams
+from SpinCore_pp.power_helper import gen_powerlist
+from pyspecdata.file_saving.hdf_save_dict_to_group import hdf_save_dict_to_group
+import h5py
+# {{{ Combined ODNP
+# {{{ experimental parameters
+# {{{ these need to change for each sample
+output_name = '150mM_TEMPOL_DNP'
+IR_postproc = 'spincore_IR_v1'
+Ep_postproc = 'ODNP_v3'
+adcOffset = 26
+carrierFreq_MHz = 14.897196
+nScans = 1
+nEchoes = 1
+# all times in microseconds
+# note that acq_time_ms is always milliseconds
+p90_us = 4.464
+repetition_us = 0.5e6
+FIR_rd = 0.5e6
+vd_list =r_[2.1e3,1.12e4,2.23e4,3.3e4,4.4e4,5.6e4,6.7e4,7.8e4,8.9e4,1e5] 
+max_power = 4 #W
+power_steps = 14
+threedown = True
+# }}}
+#{{{Power settings
+dB_settings = gen_powerlist(max_power,power_steps+1, three_down=threedown)
+T1_powers_dB = gen_powerlist(max_power, 5, three_down=False)
+T1_node_names = ['FIR_%ddBm'%j for j in T1_powers_dB]
+print("dB_settings",dB_settings)
+print("correspond to powers in Watts",10**(dB_settings/10.-3))
+print("T1_powers_dB",T1_powers_dB)
+print("correspond to powers in Watts",10**(T1_powers_dB/10.-3))
+myinput = input("Look ok?")
+if myinput.lower().startswith('n'):
+    raise ValueError("you said no!!!")
+#}}}
+powers = 1e-3*10**(dB_settings/10.)
+fl = figlist_var()
+save_file=True
+SW_kHz = 3.9 
```

Comment:
please check with @aabeaton and add a comment about what the minimum SW w/out loss is.